### DISCUSSION TITLE
Missing Documentation

### DISCUSSION THREAD

**Discussion by alilleybrinker** [mitre/hipcheck#678] [General]
We need more documentation! This is a general purpose statement that would _always_ be true, but in particular we've added a bunch of new concepts and functionality with the introduction of the plugin system, and it's not all documented clearly for users. This issue is intended to track documentation we ought to write.

- [ ] Explain targets structs and how they're passed to default queries

**Comment by ninaagrawal** [comment]
sdk/rust/src/mock.rs should have a comment at the top explaining purpose of the file. More comments throughout the file would be good

**Comment by ninaagrawal** [comment]
Giving more details about what configs are for plugins would be useful

**Comment by ninaagrawal** [comment]
For plugins/affiliation/src/until/org_types.rs, an annotation explaining what hosts are might be helpful. I think I saw some examples in the other files within the src directory but explaining what hosts are would still be good
